### DISCUSSION TITLE
OctaneEngine v3.0.0

### DISCUSSION THREAD

**Discussion by gregyjames** [gregyjames/OctaneDownloader#57] [Announcements]
## What's Changed
* Memory Optimization by @gregyjames in https://github.com/gregyjames/OctaneDownloader/pull/38
* support .net 4.6.1 and .net 4.7.2 by @andywu188 in https://github.com/gregyjames/OctaneDownloader/pull/40
* Progress fix by @gregyjames in https://github.com/gregyjames/OctaneDownloader/pull/43
* 48 throttling by @gregyjames in https://github.com/gregyjames/OctaneDownloader/pull/52
* Added Config class Support by @gregyjames in https://github.com/gregyjames/OctaneDownloader/pull/55
* 53 add proxy support by @gregyjames in https://github.com/gregyjames/OctaneDownloader/pull/56

## New Contributors
* @andywu188 made their first contribution in https://github.com/gregyjames/OctaneDownloader/pull/40

**Full Changelog**: https://github.com/gregyjames/OctaneDownloader/compare/v2.0.1...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/gregyjames/OctaneDownloader/releases/tag/v3.0.0'>OctaneEngine v3.0.0</a>.</em>

**Comment by Artemis-chan** [comment]
Is there a reason it is not pushed to nuget?

**Reply by gregyjames** [reply]
Unfortunately there was an issue with the github action responsible for pushing the package to nugget. Will upload ASAP.

**Reply by gregyjames** [reply]
3.1.0 has been pushed to Nuget.
